LayoutTests/editing/pasteboard/copy-text-from-uneditable-element.html

```diff
@@ -0,0 +1,40 @@
+<!DOCTYPE html>
+<script src="../../resources/js-test.js"></script>
+<script src="../../resources/ui-helper.js"></script>
+<button id=button>Copy foo</button>
```

Comment:
We should add another button to paste the content so that we could run this test in a browser.